core/domain/html_cleaner_test.py

```diff
@@ -608,12 +608,12 @@ def test_convert_to_ckeditor(self):
             )
         }, {
             'html_content': (
-                '<p><oppia-noninteractive-image filepath-with-value="amp;quot;'
+                '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
```

Comment:
Why was this missed last time?

Reply:
That was a mistake from side, sorry about that :sweat_smile: